console/cdvdcontrol/cdvdcontrol.cpp

```diff
@@ -104,148 +102,132 @@
+    if (!pioneer_get_quiet(dev))
       dev->ven_features |= PIO_QUIET;
-    }
   }
 
-  //  printf("Trying opcode E9 modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode E9 modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  //  printf("Trying opcode ED modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode ED modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  if ((flags & FL_SUPPORTED) != 0U) {
-    //      printf("____________________________\n");
-    cout << "\n** Supported features:\n";
-    cout << "AudioMaster Q.R.    : "
-         << (((dev->ven_features & YMH_AMQR) != 0U) ? "YES" : "-") << "\n";
+  if (flags & FL_SUPPORTED) {
+    //		printf("____________________________\n");
+    printf("\n** Supported features:\n");
+    printf("AudioMaster Q.R.    : %s\n",
+           dev->ven_features & YMH_AMQR ? "YES" : "-");
     printf("Yamaha ForceSpeed   : %s\n",
-           ((dev->ven_features & YMH_FORCESPEED) != 0U) ? "YES" : "-");
+           dev->ven_features & YMH_FORCESPEED ? "YES" : "-");
     printf("Yamaha DiscT@2      : %s\n",
-           (dev->ven_features & YMH_TATTOO) ? "YES" : "-");
+           dev->ven_features & YMH_TATTOO ? "YES" : "-");
     printf("Hide CD-R           : %s\n",
-           (dev->ven_features & PX_HCDRSS) ? "YES" : "-");
+           dev->ven_features & PX_HCDRSS ? "YES" : "-");
     printf("SingleSession       : %s\n",
-           (dev->ven_features & PX_HCDRSS) ? "YES" : "-");
+           dev->ven_features & PX_HCDRSS ? "YES" : "-");
     printf("SpeedRead           : %s\n",
-           (dev->ven_features & PX_SPDREAD) ? "YES" : "-");
+           dev->ven_features & PX_SPDREAD ? "YES" : "-");
     printf("PoweRec             : %s\n",
-           (dev->ven_features & PX_POWEREC) ? "YES" : "-");
+           dev->ven_features & PX_POWEREC ? "YES" : "-");
     printf("GigaRec             : %s\n",
-           (dev->ven_features & PX_GIGAREC) ? "YES" : "-");
+           dev->ven_features & PX_GIGAREC ? "YES" : "-");
     printf("VariRec CD          : %s\n",
-           (dev->ven_features & PX_VARIREC_CD) ? "YES" : "-");
+           dev->ven_features & PX_VARIREC_CD ? "YES" : "-");
     printf("VariRec DVD         : %s\n",
-           (dev->ven_features & PX_VARIREC_DVD) ? "YES" : "-");
+           dev->ven_features & PX_VARIREC_DVD ? "YES" : "-");
     printf("SecuRec             : %s\n",
-           (dev->ven_features & PX_SECUREC) ? "YES" : "-");
+           dev->ven_features & PX_SECUREC ? "YES" : "-");
     printf("Silent mode         : %s\n",
-           (dev->ven_features & PX_SILENT) ? "YES" : "-");
+           dev->ven_features & PX_SILENT ? "YES" : "-");
     printf("DVD+R bitsetting    : %s\n",
```

Comment:
Codacy found an issue: [Clarify calculation precedence for '&' and '?'.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6086205)